template/androidApp/src/test/java/io/bloco/template/ExampleUnitTest.kt

```diff
@@ -1,17 +1,28 @@
+
 package io.bloco.template
 
+import io.bloco.core.commons.logDebug
+import io.bloco.core.commons.setup
 import org.junit.Test
 
-import org.junit.Assert.*
+import org.junit.Assert.assertEquals
+import java.util.logging.Level
 
 /**
  * Example local unit test, which will execute on the development machine (host).
  *
  * See [testing documentation](http://d.android.com/tools/testing).
  */
 class ExampleUnitTest {
+    init {
+        io.bloco.core.commons.RootLogger.setup({ _: Level, _: String, message: String, _: Throwable? ->
+            println(message)
+        }, true)
+    }
+
     @Test
     fun addition_isCorrect() {
+        logDebug { "something" }
```

Comment:
I get that this is a test, but then write an actual test to check the RootLogger is working. But it's better to not have this at all, and have an actual unit test.